packages/create-app/src/services/init.ts

```diff
@@ -47,23 +47,14 @@ async function init(options: InitOptions) {
     await file.mkdir(templateDownloadDir)
     let tasks: ConstructorParameters<typeof ui.Listr>[0] = []
 
+    output.info(output.content`❯ Downloading template from ${repoUrl}`)
```

Comment:
The progressUpdater is pretty valuable to the UX, it's unfortunate to see it go. Is there any way to have a self-replacing output similar to what Listr provides, so we can display progress?

Reply:
Actually, log-update would allow us to start with `❯ Downloading...` and then replace it with `✓ Downloaded...` at the end. And that would look nice with Listr too!